--- a/src/lj_arch.h
+++ b/src/lj_arch.h
@@ -383,7 +383,7 @@
 #endif
 #elif LJ_TARGET_MIPS
 #if defined(__mips_soft_float)
-#error "No support for MIPS CPUs without FPU"
+#warning "No support for MIPS CPUs without FPU"
 #endif
 #if defined(_LP64)
 #error "No support for MIPS64"
diff --git a/src/luaconf.h b/src/luaconf.h
index 043590b..d5f7d19 100644
--- a/src/luaconf.h
+++ b/src/luaconf.h
@@ -35,8 +35,8 @@
 #ifndef LUA_LMULTILIB
 #define LUA_LMULTILIB  "lib"
 #endif
-#define LUA_LROOT      "/usr/local"
-#define LUA_LUADIR     "/lua/5.1/"
+#define LUA_LROOT      "/usr/"
+#define LUA_LUADIR     "/lua/"
 #define LUA_LJDIR      "/luajit-2.1.0-alpha/"
 
 #ifdef LUA_ROOT
 #define LUA_JROOT	LUA_ROOT
